examples/stabilizer_code.py

```diff
@@ -8,8 +8,65 @@
 # https://thesis.library.caltech.edu/2900/2/THESIS.pdf
 
 
-class StabilitizerCode(object):
-    def __init__(self, group_generators):
+def _BuildByCode(mat):
```

Comment:
Docstrings and type annotations for this and `_GaussianElimination` would aid readability.

Reply:
Done.